openpype/modules/kitsu/plugins/publish/collect_kitsu_username.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+import os
+import re
+
+import pyblish.api
+
+
+class CollectKitsuUsername(pyblish.api.ContextPlugin):
+    """Collect Kitsu username from the kitsu login"""
+
+    order = pyblish.api.CollectorOrder + 0.499
+    label = "Kitsu username"
+
+    def process(self, context):
+        kitsu_login = os.environ['KITSU_LOGIN']
```

Comment:
No sure if `KITSU_LOGIN` env is always filled?
```suggestion
        kitsu_login = os.environ.get('KITSU_LOGIN')
```